src/environment.cpp

```diff
@@ -426,13 +426,15 @@ bool ServerEnvironment::line_of_sight(v3f pos1, v3f pos2, float stepsize, v3s16
 	return true;
 }
 
-void ServerEnvironment::kickAllPlayers(const std::string &reason)
+void ServerEnvironment::kickAllPlayers(u8 reason,
```

Comment:
The compile time of minetest is not so huge, and we could optimize headers later, by using precompiled headers if we want to be faster in the future :)